tests/a11y-no-visually-hidden-interactive-element.js

```diff
@@ -0,0 +1,45 @@
+const rule = require('../lib/rules/a11y-no-visually-hidden-interactive-element')
+const RuleTester = require('eslint').RuleTester
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+})
+
+const errorMessage =
+  'Avoid visually hidding interactive elements. Visually hiding interactive elements can be confusing to sighted keyboard users as it appears their focus has been lost when they navigate to the hidden element.'
+
+ruleTester.run('a11y-no-visually-hidden-interactive-element', rule, {
```

Comment:
Can we add a test to make sure the options work as expected? [Here's an example](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/f0d2ddb65f21278ad29be43fb167a1092287b4b1/__tests__/src/rules/anchor-ambiguous-text-test.js#L112)! :)

Reply:
Amazing! I didn't realize that was a test option!